features/facebookImport/src/model/analyses/report/importing-status-analysys.js

```diff
@@ -7,38 +7,54 @@ export default class DataImportingStatusAnalysis extends ReportAnalysis {
     }
 
     get reportData() {
-        return this._importingResults.map(
-            ({ status, importerClass, message }) => {
-                return {
-                    status,
-                    importerName: importerClass.name,
-                    message,
-                };
-            }
-        );
+        return this._importersData;
     }
 
     async analyze({ facebookAccount }) {
-        this._importingResults = facebookAccount.importingResults;
-        this.active = this._importingResults.length > 0;
+        this._importersData = facebookAccount.importingResults.map(
+            (importerResult) => importerResult.reportJsonData
+        );
+        this.active = this._importersData.length > 0;
+    }
+
+    _renderStatus(status) {
+        return (Array.isArray(status) ? status : [status]).map(
+            (each, statusIndex) => {
+                return (
+                    <div key={statusIndex}>
+                        {each.name + (each.message ? " - " + each.message : "")}
+                    </div>
+                );
+            }
+        );
     }
 
     render() {
         return (
             <>
                 <p>
-                    Data was read using {this._importingResults.length}{" "}
-                    importers. This is a technical view showing the list of
-                    importers that read data.
+                    Data was read using {this._importersData.length} importers.
+                    This is a technical view showing the list of importers that
```

Comment:
Grammar nit: what do you mean by "technical view"?

Reply:
Mostly that is mostly for us and not the user. I can remove "technical" as it does not bring much value.

Reply:
In that sense, just "this is the list of importer that read data" would be more than enough...

Reply:
I changed it before seeing this message to almost the exact same thing!